composer.json

```diff
@@ -7,14 +7,14 @@
         "symfony/debug": "^2.6|^3.0",
         "symfony/process": "^2.6|^3.0",
         "react/event-loop": "^0.4",
-        "react/stream": "^0.4",
-        "react/http": "dev-master#cd15204bd15d106d7832c680e4fb0ca0ce2f5e30",
-        "react/socket-client": "^0.5.0",
-        "react/socket": "^0.5.0",
+        "react/stream": "^0.6",
+        "react/http": "dev-master",
+        "react/socket-client": "^0.6",
+        "react/socket": "^0.5|^0.6|^0.7",
         "mkraemer/react-pcntl": "2.0.*",
         "monolog/monolog": "^1.3",
         "paragonie/random_compat": "^2",
-        "guzzlehttp/psr7": "^1.2"
```

Comment:
Shouldn't we use ringcentral here as long as we're directly having dependencies (i.e. React/HeaderParser)?